auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -28,26 +23,19 @@
       <v-row no-gutters id="dashboard-actions" class="mb-n3">
         <v-col cols="auto">
           <!-- Add Existing Name Request or Business -->
-          <v-menu
-            v-model="addAffiliationDropdown"
-          >
+          <v-menu v-model="addAffiliationDropdown">
             <template v-slot:activator="{ on: onMenu }">
               <v-tooltip top content-class="top-tooltip">
                 <template v-slot:activator="{ on: onTooltip }">
-                  <v-btn
-                    id="add-existing-btn"
-                    class="mt-2 mr-4"
-                    color="primary"
-                    dark large
-                    v-on="{ ...onMenu, ...onTooltip }"
-                    @click="addAffiliationDropdown = !addAffiliationDropdown"
-                  >
+                  <v-btn id="add-existing-btn" class="mt-2 mr-4" color="primary" dark large
+                    v-on="{ ...onMenu, ...onTooltip }" @click="addAffiliationDropdown = !addAffiliationDropdown">
                     <v-icon>mdi-plus</v-icon>
                     <span><strong>Add an Existing Business or Name Request</strong></span>
                     <v-icon class="ml-2 mr-n2">{{ addAffiliationDropdown ? 'mdi-menu-up' : 'mdi-menu-down' }}</v-icon>
                   </v-btn>
                 </template>
-                <span>To view and manage existing businesses and Name Requests, you can manually add them to your table.</span>
+                <span>To view and manage existing businesses and Name Requests, you can manually add them to your
+                  table.</span>
```

Comment:
Spans are inline elements (not block level) so this change injects a carriage return into the string. This may be a problem in unit tests that check for exact matches, or may result in a newline on screen depending how this span is used.

Also, our max line length is 120, although I generally try to keep them under 100 as you have done here.

I'm not saying you need to revert this, but you do need to aware. If this still displays OK then this is fine.